python/lumifit/reconstruction.py

```diff
@@ -138,13 +138,13 @@ def create_reconstruction_job(
         resource_request = make_test_job_resource_request()
 
     application_command = (
-        f"{lmdScriptPath}/singularityJob.sh {lmdScriptPath}/runLmdReco.py"
```

Comment:
we shouldn't hardcode this at all, there needs to be a reference to the config here